lib/telegraf.js

```diff
@@ -119,7 +119,7 @@ class Telegraf extends Composer {
     }
     this.telegram.getUpdates(timeout, limit, offset, allowedUpdates)
       .catch((err) => {
-        const wait = err.retryAfter || this.options.retryAfter
+        const wait = err.parameters.retry_after || this.options.retryAfter
```

Comment:
TelegramError.parameters always contains an object
`this.parameters = payload.parameters || {}`